### PR TITLE
Chore/introduce self service gen3 roll

### PR DESCRIPTION
**Pull request by themarcelor**
This script should help DEVs / Bioinformaticians run `gen3 roll all` against any k8s namespace in qaplanetv2:
https://jenkins2.planx-pla.net/job/self-service-qa-gen3-roll/

### CODE REVIEW

File: scripts/self_service_gen3_roll.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const { Bash } = require('../utils/bash');
+
+const bash = new Bash();
+
+async function runGen3Roll(serviceName) {
+  process.env['KUBECTL_NAMESPACE'] = process.env.TARGET_ENVIRONMENT;
+  process.env['GEN3_HOME'] = `${require('path').resolve(__dirname, '../..')}/cloud-automation`;
+  const debug1 = await  bash.runCommand('ls && pwd && cd .. && ls -ilha && pwd && ls && cd .. && ls');
+  console.log(`result: ${debug1}`);
+  const cmd1 = await  bash.runCommand(`export KUBECTL_NAMESPACE=${process.env.TARGET_ENVIRONMENT} && gen3 kube-setup-secrets && gen3 roll ${serviceName}`);
```
Comment (atharvar28) [inline scripts/self_service_gen3_roll.js:10]: spaces before the `bash.runCommand`

Comment (themarcelor) [reply scripts/self_service_gen3_roll.js:10]: 🤦 ugh, sorry. Lemme fix that.

File: scripts/self_service_gen3_roll.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const { Bash } = require('../utils/bash');
+
+const bash = new Bash();
+
+async function runGen3Roll(serviceName) {
+  process.env['KUBECTL_NAMESPACE'] = process.env.TARGET_ENVIRONMENT;
+  process.env['GEN3_HOME'] = `${require('path').resolve(__dirname, '../..')}/cloud-automation`;
+  const debug1 = await  bash.runCommand('ls && pwd && cd .. && ls -ilha && pwd && ls && cd .. && ls');
```
Comment (atharvar28) [inline scripts/self_service_gen3_roll.js:8]: here as well @themarcelor

**Comment by themarcelor** [comment]
This is no longer necessary.

**Pull request closed without merging**